presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastore.java

```diff
@@ -69,6 +69,17 @@ public void setUp()
         stats = thriftHiveMetastore.getStats();
     }
 
+    private ListeningExecutorService createTestThreadPool()
```

Comment:
undo, it is used only in single place

Reply:
in 2 places

Reply:
Right, I forgot to comment the other usage ;)